src/main/java/com/github/krzysiekjodlowski/ox/Event.java

```diff
@@ -0,0 +1,16 @@
+package com.github.krzysiekjodlowski.ox;
+
+/**
+ * Describes generic event, and it's associated meta data which get
+ * sent in the bus to be dispatched to interested Subscribers.
```

Comment:
I think send "via" or "by" but not "in the", though I am not 100% sure

Reply:
Your'e right, aah these ready solutions and their english:P